README.md

```diff
@@ -0,0 +1,47 @@
+# seniormap
+An interactive map of GMHS seniors' college/postsecondary destinations.  Created by Erik Boesen (GMHS Class of 2019), the map is now maintained the GMHS Advanced Programming club. If you'd like to make a submission to the map please navigate to https://apc-gm.com/seniormap/ and click on the form in the bottom right corner. 
+
+## Maintainence instructions 
+### Creating a the form and spreadsheet for a new year
+
+1. Navigate to the 'Senior Map' folder on Google Drive and create a new folder with the year of the graduating class (e.g. 2027)
+2. Make a copy of a previous year's form and edit content accordingly for the appropriate year
+3. Ensure the spreadsheet has a [linked Google spreadsheet](https://drive.google.com/file/d/1aFtfqwHGrV9CzMXrnqTfLghL3VVLddmU/view?usp=sharing) with appropriate fields
+
+### Adding the new year to the code base
+
+1. Within `js/scripts.js`, find the `var dataDocuments` and add an entry, with the spreadsheet's datasheet URL for the appropriate year 
+  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For example: `['2027', 'a8a8a8a8a8a8a8a8a8a8a8a8a8a'],`
+
+2. Within `index.html` update the linked form link to the latest year 
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For example: `<p><a href="https://forms.gle/form">Submit your destination here</a></p>`
+
+3. Create an entry mimcking the existing entry cells for the spreadsheet's datasheet URL in the Google Drive's 'Year Data' folder (Senior Map/Shared Data/Year Data)
+
+### Obtaining and readying senior portraits of students
+
+1. Email the registrar for a list of student. He/she for them will provide them to you in spreadsheet form
```

Comment:
I've included a link to the admin staff directory and I'll put the current registrar's name as well. Knowing GM, they'll switch up their website next week, and replace the registrar next month, so this will probably have to be updated in the future.